tests/components/downloader/test_init.py

```diff
@@ -1,29 +1,15 @@
 """Tests for the downloader component init."""
 
-from unittest.mock import patch
+import pytest
 
-from homeassistant.components.downloader.const import (
-    CONF_DOWNLOAD_DIR,
-    DOMAIN,
-    SERVICE_DOWNLOAD_FILE,
-)
+from homeassistant.components.downloader.const import DOMAIN, SERVICE_DOWNLOAD_FILE
 from homeassistant.config_entries import ConfigEntryState
 from homeassistant.core import HomeAssistant
 
-from tests.common import MockConfigEntry
 
-
-async def test_initialization(hass: HomeAssistant) -> None:
+@pytest.mark.usefixtures("loaded_config_entry")
+async def test_initialization(hass: HomeAssistant, loaded_config_entry) -> None:
     """Test the initialization of the downloader component."""
-    config_entry = MockConfigEntry(
-        domain=DOMAIN,
-        data={
-            CONF_DOWNLOAD_DIR: "/test_dir",
-        },
-    )
-    config_entry.add_to_hass(hass)
-    with patch("os.path.isdir", return_value=True):
-        assert await hass.config_entries.async_setup(config_entry.entry_id)
 
     assert hass.services.has_service(DOMAIN, SERVICE_DOWNLOAD_FILE)
-    assert config_entry.state is ConfigEntryState.LOADED
+    assert loaded_config_entry.state is ConfigEntryState.LOADED
```

Comment:
`assert hass.services.has_service(DOMAIN, SERVICE_DOWNLOAD_FILE)` and `assert loaded_config_entry.state is ConfigEntryState.LOADED` implies the previous `assert await hass.config_entries.async_setup(config_entry.entry_id)`, correct?